github/authorizations.go

```diff
@@ -302,10 +306,14 @@ func (s *AuthorizationsService) Check(ctx context.Context, clientID string, toke
 // The returned Authorization.User field will be populated.
 //
 // GitHub API docs: https://developer.github.com/v3/oauth_authorizations/#reset-an-authorization
```

Comment:
This URL should now be: https://developer.github.com/v3/apps/oauth_applications/#reset-a-token

Also, the comment starting above around line 302 is no longer valid. Please update it.